src/App.css

```diff
@@ -1,3 +1,31 @@
+:root{
+
+  --pri-text-color:rgb(39, 39, 43) ;
+  --pri-color:#FFCC00;
+  
+
+}
+.pri-text-color{
+  color: var(--pri-text-color);
+}
+.pri-color{
+  color:var(--pri-color);
+}
+.pri-bg-color{
+  background-color: var(--pri-color);
+}
+.text-center{
+  text-align: center;
+}
+.gap-sm{
```

Comment:
You must be having these utility classes in the component library.Can import the same and use it directly

Reply:
yeah , but i forgot these things in CL ( maybe in future refactore and do it ) thanks for suggesting